.github/workflows/nightly.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020-2021 Dave Verwer, Sven A. Schmidt, and other contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Query Performance Test
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
To do: add `schedule` instead of `push` after testing is complete

Reply:
This runs faster than the regular CI and I think it makes more sense to run it in parallel in fact. If there are changes being introduced that cause a performance regression we'll want to find out while the PR is up and not have to figure out what it was a day later.